pkg/runtime/conversion_generator.go

```diff
@@ -24,12 +24,13 @@ import (
 	"sort"
 	"strings"
 
+	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/conversion"
 	"k8s.io/kubernetes/pkg/util/sets"
 )
 
 type ConversionGenerator interface {
-	GenerateConversionsForType(version string, reflection reflect.Type) error
+	GenerateConversionsForType(groupVersion unversioned.GroupVersion, reflection reflect.Type) error
```

Comment:
this is an example of where we are inconsistent in this PR.

groupVersion unversioned.GroupVersion

and later:

gv unversioned.GroupVersion